make_wheels.py

```diff
@@ -1,3 +1,10 @@
+# /// script
+# requires-python = ">=3.9"
```

Comment:
I added this as 3.9. We can leave it as 3.8, too, but I'd recommend not to do so because Python 3.8 is EoL now (same suggestion goes for `pyproject.toml`, where it's currently 3.8).

Reply:
Sure. It looks like [pdm can use this metadata too](https://github.com/pdm-project/pdm/pull/2924/files). Shall we switch to PEP 723 only?

Reply:
Ah, I guess that wouldn't install `twine`...

Reply:
Yes, I tried that too, too bad we can't add `dev` dependencies. But we can still make the switch to and make `twine` a "required" dependency:
- it's small (38.6 KiB for the latest release),
- `wheel` and `twine` don't/won't depend on one another – no conflicts expected,
- `twine` is not used in the script directly, and
- anyone using `twine` without tokens won't be able to upload wheels to PyPI anyway

What do you think?

Reply:
Doesn't `twine` pull in like a dozen other packages?

In any case it's probably fine, I can just `pipx install twine`.

Reply:
> Doesn't `twine` pull in like a dozen other packages?

Ah, you're right, forgot about that. Based on the dependencies from https://www.wheelodex.org/projects/twine/, besides `requests` and `urllib3`, none of us are useful for us (except, perhaps, `pkginfo` and `readme-renderer` for validation).

I don't see any similar asks for in projects that have implemented the PEP, maybe because no one would want a dependency if it isn't being imported in the script.


> In any case it's probably fine, I can just `pipx install twine`.

Yes. Another way is `uv run --with twine make_wheels.py`, where `uv` can both the script metadata and installs `twine` with it. This option isn't present in `pipx`, IIUC.

I'll delete the `pyproject.toml` file, then, and add some additional commands to the README.

Reply:
Addressed, apologies for the force push.

Reply:
No need to apologize, I prefer force pushes.